base/config/base.json

```diff
@@ -1,4 +1,5 @@
 {
     "logs_directory": "base/log",
-    "shutdown_between_backups": true
+    "shutdown_between_backups": true,
+    "email_notification_receivers": ["maxiwelsch@posteo.de"]
```

Comment:
Ich würde meine Mail-Adresse nicht in ein öffentliches Repo schreiben!
Außerdem sollten Benutzerspezifische Daten eh nicht ins Repo.

Reply:
Um das wieder loszuwerden, empfehle ich einen interactive rebase wie hier beschrieben: https://www.bryanbraun.com/2019/02/23/editing-a-commit-in-an-interactive-rebase/ und dann einen force push auf origin: https://fedingo.com/how-to-delete-commits-in-git/

Reply:
Prinzipiell hast Du Recht.